src/components/GSelect/GCombobox.vue

```diff
@@ -0,0 +1,392 @@
+        if (props.chips && selections.value) {
+          return <GChip small={props.smallChips} close={props.deletableChips}
+                        vOn:close={() => onChipCloseClick()}>{selections.value}</GChip>
+        }
+      }
+
+      const selectionsText = computed(() => {
+        return props.multiple ? selections.value.join('') : selections.value
+      })
+      //textfield logic, styles, classes computed
+      const isValidInput = ref(true)
+      const isFocused = ref(false);
+      const validateText = computed(() => lazySearch.value || selectionsText.value || state.searchText)
+      const {labelClasses, labelStyles, isDirty, isLabelActive, prefixRef} = getLabel(context, props, validateText, isValidInput, isFocused, 'g-tf-label__active');
+      const hintClasses = computed(() => (props.persistent || (isFocused.value && isValidInput.value)) ? {'g-tf-hint__active': true} : {})
+      const {errorMessages, validate} = getValidate(props, isFocused, validateText, isValidInput);
+
+      //textfield events
+      const inputAddSelection = () => {
+        if (state.searchText.trim().length !== 0) {
+          let inputAddedItem
+          props.itemValue
+              ? inputAddedItem = {
+                [props.itemText]: state.searchText,
+                [props.itemValue]: state.searchText
+              } :
+              inputAddedItem = {
+                [props.itemText]: state.searchText
+              }
+          toggleItem(inputAddedItem)
+          setSearch()
+        }
+      }
+      function clearSelection() {
+        selectedItem.value = props.multiple ? [] : ''
+        setSearch()
+      }
+      function onInputKeyDown(e) {
+        setSelectionsDisplay
+        if (e.keyCode === keyCodes.down) {
+          context.root.$el.getElementsByClassName('g-list-item')[0].focus()
+        }
+      }
+      function onInputClick() {
+        isFocused.value = true
+        state.searchText = ''
+      }
+      function onInputBlur() {
+        isFocused.value = false
+        setSelectionsDisplay
+      }
+
+      let pressDeleteTime = 0
+      const deleteItemColor = ref('#1d1d1d')
+      const setSelectionsDisplay = () => {
+        pressDeleteTime = 0
+        deleteItemColor.value = '#1d1d1d'
+      }
+      function onInputDelete() {
+        if (!props.multiple || props.chips) return
+        if (state.searchText) return pressDeleteTime = 0
+        else {
+          if (pressDeleteTime === 0) {
+            pressDeleteTime++
+            deleteItemColor.value = '#1867c0 '
+          }
+          if (pressDeleteTime === 1) {
+            return pressDeleteTime++
+          }
+          if (pressDeleteTime === 2) {
+            selectedItem.value.pop()
+            return pressDeleteTime
+          }
+        }
+      }
+
+      const tfValue = computed(() =>
+          props.multiple ? state.searchText : lazySearch.value)
+
+      //textfield slots
+      const getTextFieldScopedSlots = {
```

Comment:
getTextFieldScopedSlots -> textFieldScopedSlots (variable, not named function)
move variable into render function's scope